app/controllers/businesses_controller.rb

```diff
@@ -2,10 +2,8 @@ class BusinessesController < ApplicationController
 
   def index
     @businesses = Business.all
-    @categories = Category.all
```

Comment:
we will need to put this line back in, I'll need it to be able to build the Categories dropdown.